seth/gas_adjuster.go

```diff
@@ -491,15 +483,15 @@ func (m *Client) HistoricalFeeData(priority string) (baseFee float64, historical
 		historicalGasTipCap = stats.TipCap.Perc25
 	default:
 		err = fmt.Errorf("unknown priority: %s", priority)
-		L.Error().
+		L.Debug().
 			Str("Priority", priority).
-			Msg("Unknown priority. Skipping automation gas estimation")
+			Msgf("Unknown priority: %s. Skipping automation gas estimation", err.Error())
```

Comment:
Can we enforce it by type so you can't insert a "string"?

Reply:
we can. but would need to experiment how that would work with TOML